.github/workflows/version.yml

```diff
@@ -33,9 +33,14 @@ jobs:
               run: git checkout -b version-bump-${{ github.sha }}
 
             - name: Generate version
+              id: generate-new-version
               uses: Expensify/Expensify.cash/.github/actions/bumpVersion@master
               with:
                   GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
+                  SEMVER_LEVEL: PATCH
+            
+            - name: Print new version
```

Comment:
I know you probably just had this for testing, but we won't need it in production.

Reply:
I was not sure about what to do here. Should I just remove the SEMVER_LEVEL (and also the Print new version?)

Reply:
Oh, you should definitely keep the `SEMVER_LEVEL`, but we'll merge it w/ `SEMVER_LEVEL: BUILD`.

And you can leave this for now, but right before merging we can remove the `Print new version` step.